docs/en_US/Blog/HPOComparison.md

```diff
@@ -0,0 +1,323 @@
+# Hyperparameter Optimization Comparison
```

Comment:
can you add more reference link and explain detail about RockDB benchmark?

Reply:
OK.